core/storage/user/gae_models_test.py

```diff
@@ -347,3 +392,48 @@ def test_get_categories_where_user_can_review(self):
         self.assertIn('category1', score_categories)
         self.assertIn('category3', score_categories)
         self.assertNotIn('category2', score_categories)
+
+
+class UserSubscriptionsModelTests(test_utils.GenericTestBase):
+    """Tests for UserSubscriptionsModel."""
+    USER_ID_1 = 'user_id_1'
+    USER_ID_2 = 'user_id_2'
+    TEST_VALUE = '1'
+
+    def setUp(self):
+        """Set up user models in datastore for use in testing."""
+        super(UserSubscriptionsModelTests, self).setUp()
+        user_models.UserSubscriptionsModel(id=self.USER_ID_1).put()
+
+        user_model_2 = user_models.UserSubscriptionsModel(id=self.USER_ID_2)
+        user_model_2.creator_ids.append(self.TEST_VALUE)
```

Comment:
You should probably use different values for each of these ('sample_creator_id', 'sample_collection_id', etc.) to make sure that the right stuff is exported to the right fields.

Reply:
@ulloaluis

Reply:
Addressed and pushed to your takeout branch @varun-tandon 👍